staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -172,6 +179,67 @@ var (
 		currentInflightRequests,
 		requestTerminationsTotal,
 	}
+
+	// these are the known (e.g. whitelisted/known) content types which we will report for
+	// request metrics. Any other RFC compliant content types will be aggregated under 'unknown'
+	knownMetricContentTypes = utilsets.NewString(
```

Comment:
is there a reason to report content types other than what we serve?

Reply:
In general, out of a sense of caution about what people may expect.